test/e2e/conftest.py

```diff
@@ -44,8 +90,259 @@ def pytest_collection_modifyitems(config, items):
 # Provide a k8s client to interact with the integration test cluster
 @pytest.fixture(scope='class')
 def k8s_client():
-    return k8s._get_k8s_api_client()
+    return k8sclient._get_k8s_api_client()
```

Comment:
Just for the sake of consistency with the other tests, would you mind changing this back to `k8s`. In conjunction with my comments below, this shouldn't conflict.